Library/Homebrew/metafiles.rb

```diff
@@ -1,13 +1,15 @@
+require "set"
+
 class Metafiles
   # https://github.com/github/markup#markups
   EXTENSIONS = %w[
     .adoc .asc .asciidoc .creole .html .markdown .md .mdown .mediawiki .mkdn
     .org .pod .rdoc .rst .rtf .textile .txt .wiki
-  ].freeze
+  ].to_set.freeze
```

Comment:
O(1) lookup versus O(n). If order doesn't matter and there are no duplicates, why use an array when you could use a set?